packages/objects/src/businessObjects/Questionnaire.ts

```diff
@@ -18,6 +18,12 @@ export enum EQuestionnaireQuestionType {
   Numeric = "numeric",
 }
 
+export enum EQuestionnaireQuestionDisplayType {
+  Dropdown = "dropdown",
+  List = "list",
+  Scale = "scale",
+}
+
```

Comment:
For `EQuestionnaireQuestionType` and `EQuestionnaireQuestionDisplayType` could you make both sides of the enum identical?

Dropdown = "Dropdown" instead of "dropdown" and Location = "Location"?

It makes it much easier to go from string to Enum and back and forth - which we have to do a bunch on the IP side

Reply:
also for Questionnaire.image, should that be a URLString or an ipfsCID?